pkg/utils/release/controller.go

```diff
@@ -39,50 +41,51 @@ func (s *SuiteController) CreateApplicationSnapshot(name string, namespace strin
 }
 
 // CreateRelease creates a new Release using the given parameters.
-func (s *SuiteController) CreateRelease(name, namespace, snapshot, sourceReleaseLink string) (*v1alpha1.Release, error) {
-	release := &v1alpha1.Release{
+func (s *SuiteController) CreateRelease(name, namespace, snapshot, sourceReleasePlan string) (*appstudiov1alpha1.Release, error) {
+	release := &appstudiov1alpha1.Release{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: namespace,
 		},
-		Spec: v1alpha1.ReleaseSpec{
+		Spec: appstudiov1alpha1.ReleaseSpec{
 			ApplicationSnapshot: snapshot,
-			ReleaseLink:         sourceReleaseLink,
+			ReleasePlan:         sourceReleasePlan,
 		},
 	}
 
 	return release, s.KubeRest().Create(context.TODO(), release)
 }
 
-// CreateReleaseLink creates a new ReleaseLink using the given parameters.
-func (s *SuiteController) CreateReleaseLink(name, namespace, application, targetNamespace, releaseStrategy string) (*v1alpha1.ReleaseLink, error) {
-	releaseLink := &v1alpha1.ReleaseLink{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      name,
-			Namespace: namespace,
-		},
-		Spec: v1alpha1.ReleaseLinkSpec{
-			DisplayName:     name,
-			Application:     application,
-			Target:          targetNamespace,
-			ReleaseStrategy: releaseStrategy,
-		},
-	}
+// GetRelease returns the release with the given name in the given namespace.
+func (s *SuiteController) GetRelease(releaseName, releaseNamespace string) (*appstudiov1alpha1.Release, error) {
```

Comment:
I think this should just be called namespace, like you have it for the ReleasePlan functions

Reply:
Fixed